test/gtest/ucp/test_ucp_wfe.cc

```diff
@@ -0,0 +1,102 @@
+/**
+* Copyright (C) Mellanox Technologies Ltd. 2001-2014.  ALL RIGHTS RESERVED.
+*
+* Copyright (C) UT-Battelle, LLC. 2015. ALL RIGHTS RESERVED.
+* See file LICENSE for terms.
+*/
+
+#include "ucp_test.h"
+
+#include <gtest/common/test_perf.h>
+
+
+#define MB   pow(1024.0, -2)
+#define UCP_ARM_PERF_TEST_MULTIPLIER 2
+class test_ucp_wfe : public ucp_test, public test_perf {
+public:
+    static void get_test_variants(std::vector<ucp_test_variant>& variants) {
```

Comment:
`std::vector<ucp_test_variant> &variants`

Reply:
I don't think this aligns with the format that is used in other instances https://github.com/openucx/ucx/blob/f473cd28a1e3e7d691fbaa6d27acafa05396d444/test/gtest/ucp/test_ucp_wireup.cc#L24

Generally speaking I have no religion about this but if I change it will be inconsistent with other instances.

Reply:
not a problem, I think we need to clarify what's suitable in UCX